traveler-ios-ui/TravelerKitUI/ModelExtensions/CancellationError+LocalizedError.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CancellationError+LocalizedError.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-07.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+extension CancellationError: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .expiredQuote:
+            return NSLocalizedString("Quote has expired", comment: "CancellationError")
+        case .notCancellable:
+            return NSLocalizedString("Quote not cancellable", comment: "CancellationError")
```

Comment:
+1

This is cool

just wanted to add that its not the "quote" that is not cancellable,...its the "order",...but thats just copy and were not perfecting copy at this time